data/gmh/Cramer_1971_Lohengrin/metadata.txt

```diff
@@ -0,0 +1,62 @@
+publicationType : "book"
+publicationTitle : "Lohengrin. Edition und Untersuchungen"
+publicationDate : "1971"
+publicationPlaces:
+    - publicationPlace: "München"
+publicationSeries : ""
+publicationNum: ""
+publicationStemmaNum: "2"
+publicationAuthors:
+    - publicationAuthor: "Cramer, Thomas"
+publicationPage : "18"
+publicationLink : " https://katalog.ub.uni-heidelberg.de/titel/1126165 "
+workTitle : "Lohengrin"
+workViaf : "181484006"
+workOrigDate : "1283-1289"
+workOrigPlace : ""
+workAuthors:
+    - workAuthor: ""
+      workAuthorViaf: ""
+workGenre : ""
+workLangCode : "gmh"
+stemmaType : "reconstructed"
+drawnStemma : "true"
+contam : "no"
+extraStemmContam : "no"
+rootType : "unspecified"
+completeWits : "included"
+sourceText : "NA"
+derivatives : "NA"
+contributors:
+    - contributor: "Johannes Barnsteiner"
+      contributorORCID: ""
+note : "The text is related to other two different works: »Wartburgkrieg« and Ulrich Fuetrer's »Buch der Abenteuer'«, which appear in the printed stemma but were not encoded."

```

Comment:
Why not ?

Reply:
Because they are influences and not really versions of the text.